packages/wells/src/client/cogniteClient.ts

```diff
@@ -35,3 +26,19 @@ export default class CogniteClient extends CogniteClientStable {
     return `wells/${version}`;
   }
 }
+
+export function setupClient(baseUrl: string = Constants.BASE_URL) {
+  return new CogniteClient({
+    appId: `${name}`,
+    baseUrl,
+  });
+}
+
+export function setupLoggedInClient() {
```

Comment:
Did this want to be removed, and the imports in the tests reverted back to `./testUtils`?

Reply:
Yea, I think they want want to create the `CogniteClient` manually. It was only for the test setup